src/FixedDataTableStore.js

```diff
@@ -11,9 +11,20 @@
 
 'use strict';
 
-import reducers from 'reducers'
-import { createStore } from 'redux'
+import reducers from 'reducers';
+import { createStore } from 'redux';
+
+const store = createStore(reducers);
+
+// Todo: Change name of getState to something relevant
+export const getState = () => {
+  const state = store.getState();
+  return {
+    maxScrollX: state.maxScrollX,
+    scrollX: state.scrollX
+  };
+};
```

Comment:
While scrolling in ReorderHandle, it was skipping some updates due to shouldComponentUpdate of FixedDataTableCell.js. So, Introduced this helper method to get the correct snapshot of required values form state.